src/components/20-molecules/file-upload/story.js

```diff
@@ -51,6 +51,10 @@ export const FileUpload = () => {
   );
   const deleteStatusText = text('deleteStatusText', 'Delete');
   const addStatusText = text('addStatusText', 'Add more');
+  const acceptedFileTypes = text(
+    'allowedFileTypes',
```

Comment:
is `allowedFileTypes` a valid file type? I think you can delete this line.

Reply:
I mixed some variable names up 😅 

Thanks for noticing and reporting.